mne/report/report.py

```diff
@@ -278,6 +279,41 @@ class _ContentElement:
     html: str
 
 
+def _check_tags(tags) -> Tuple[str]:
+    # Must be iterable, but not a string
+    if (isinstance(tags, str) or not isinstance(tags, (Sequence, np.ndarray))):
+        raise TypeError(
+            f'tags must be a collection of str, but got {type(tags)} '
+            f'instead: {tags}'
+        )
```

Comment:
This should be `_check_option('tags', tags, (str, 'array-like', Sequence))` or so (and I'm not sure you need the `Sequence` because maybe it's already handled by 'array-like'?

Reply:
It's more complicated than that, I need to check raise

- if it **is** a string (here `_check_option()` could be used), or
- if it **is not** `array-like` (which I cannot test with `_check_option()` as it will raise if the condition is not met)

Looking through `check.py` now to see if there are some other functions I could use …